libraries/RC_Channel/RC_Channel.h

```diff
@@ -144,7 +145,9 @@ class RC_Channel {
         AVOID_PROXIMITY =     40, // enable object avoidance using proximity sensors (ie. horizontal lidar)
         ARMDISARM_UNUSED =    41, // UNUSED
         SMART_RTL =           42, // change to SmartRTL flight mode
+#if AP_INVERTED_FLIGHT_ENABLED
```

Comment:
This is the first example of using a define to remove something from the enum. This would start a new pattern. 

I do think it is a good idea. The "KILL_IMU" option would be a good candidate too.